src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -1,23 +1,37 @@
 package se.sundsvall.seabloader.service;
 
-import static java.util.Objects.nonNull;
-import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoiceEntity;
-
+import org.apache.commons.collections4.ListUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
 import se.sundsvall.seabloader.integration.db.InvoiceRepository;
+import se.sundsvall.seabloader.integration.db.model.enums.Status;
+import se.sundsvall.seabloader.integration.invoicecache.InvoiceCacheClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.FAILED;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.PROCESSED;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.UNPROCESSED;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInExchangeInvoice;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoiceEntity;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoicePdfRequest;
 
 @Service
 public class InvoiceService {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(InvoiceService.class);
+	private static final Status[] STATUSES_TO_SEND = {UNPROCESSED, FAILED};
```

Comment:
I think there was discussion of having a separate status for incoming requests that failed to be processed and for those that failed when sending them to invoice cache, right? Now all failing entities will be retried and sent to invoice cache, but those that was unprocessable upon receival will never be successfully sent to cache.

TLDR; Add new status, EXPORT_FAILED, which will be set when export is not successful. Replace FAILED with the new status in the list above.

Reply:
I've missed this discussion. I'll add the new status.